app/services/npq/application/change_funded_place.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module NPQ
+  module Application
+    class ChangeFundedPlace
+      include ActiveModel::Model
+      include ActiveModel::Attributes
+
+      attribute :npq_application
+      attribute :funded_place
+
+      validates :npq_application, presence: { message: I18n.t("npq_application.missing_npq_application") }
+      validate :accepted_application
+      validate :eligible_for_funding
+      validate :eligible_for_removing_funding_place
+      # validate :funding_cap_available
+
+      def call
+        return self unless valid?
+
+        ApplicationRecord.transaction do
+          if FeatureFlag.active?("npq_capping")
```

Comment:
maybe the feature flag should be at the beginning of the call method to avoid even validation?